### PR TITLE
refactor: migrate nft plugin

### PR DESCRIPTION
**Pull request by guanbinrui**
Part of #3066.

### CODE REVIEW

File: packages/maskbook/src/plugins/NFT/index.ts [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import { registerPlugin } from '@dimensiondev/mask-plugin-infra'
+import { base } from './base'
+
+registerPlugin({
+    ...base,
+    SNSAdaptor: {
+        load: () => import('./SNSAdaptor'),
+        hotModuleReload: (hot) =>
+            import.meta.webpackHot &&
+            import.meta.webpackHot.accept('./SNSAdaptor/index', () => hot(import('./SNSAdaptor'))),
```
Comment (septs) [inline packages/maskbook/src/plugins/NFT/index.ts:10]:
```suggestion
            import.meta.webpackHot?.accept('./SNSAdaptor/index', () => hot(import('./SNSAdaptor'))),
```

File: packages/maskbook/src/plugins/NFT/Worker/index.ts [hunk 2]
```diff
@@ -0,0 +1,8 @@
+import type { Plugin } from '@dimensiondev/mask-plugin-infra'
+import { base } from '../base'
+
```
Comment (Jack-Works) [inline packages/maskbook/src/plugins/NFT/Worker/index.ts:3]: It seems this Worker is not doing anything. Did you forget to import its RPC to start it in the Worker?

**Review by Jack-Works — CHANGES_REQUESTED** [review]

File: packages/maskbook/src/plugins/NFT/Worker/index.ts (hunk 2 above)
Comment (guanbinrui) [reply packages/maskbook/src/plugins/NFT/Worker/index.ts:3]: Removed.

Comment (Jack-Works) [reply packages/maskbook/src/plugins/NFT/Worker/index.ts:3]: did you commit?

**Review by Jack-Works — APPROVED** [review]

**Pull request merged**